pkg/grafana/grafana.go

```diff
@@ -34,10 +76,8 @@ func CreateOrganization(ctx context.Context, grafanaAPI *client.GrafanaHTTPAPI,
 	}
 	logger.Info("created organization")
 
-	return Organization{
-		ID:   *createdOrg.Payload.OrgID,
-		Name: organization.Name,
-	}, nil
+	organization.ID = *createdOrg.Payload.OrgID
+	return organization, nil
 }
 
 func UpdateOrganization(ctx context.Context, grafanaAPI *client.GrafanaHTTPAPI, organization Organization) (Organization, error) {
```

Comment:
Same remark as `CreateOrganization` regarding function signature and returns.

Reply:
done